support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/DigitalSubscriptionDirectPurchaseBuilder.scala

```diff
@@ -49,4 +54,22 @@ class DigitalSubscriptionDirectPurchaseBuilder(
 
   }
 
+  def overridePricingIfRequired(product: DigitalPack, maybeAbTests: Option[Set[AbTest]]) =
+    if (isValidBenefitsTestPurchase(product, maybeAbTests)) {
+      product.amount
+        .map { amount =>
+          val ratePlanChargeId =
+            if (product.billingPeriod == Monthly) config.monthlyChargeId else config.annualChargeId
```

Comment:
so we have a new "monthly" and "annual" digital sub charge under the same rateplan, which we use for setting the charge of non standard priced digital subs, correct?

Reply:
It is the same charge as we use for all monthly & annual digisubs currently, we're just overriding the amount.